ocs_ci/ocs/node.py

```diff
@@ -301,6 +301,52 @@ def add_new_node_and_label_it(machineset_name):
     return new_spun_node[0]
 
 
+def add_new_node_and_label_upi(node_type, num_nodes, mark_for_ocs_label=True):
+    """
+    Add a new node for aws upi platform and label it
+
+    Args:
+        node_type (str): Type of node, RHEL or RHCOS
+        num_nodes (int): number of nodes to add
+        mark_for_ocs_label (bool): True if label the new node
+    Retuns:
+        bool: True if node addition has done successfully
+    """
+
+    initial_nodes = tests.helpers.get_worker_nodes()
+    from ocs_ci.ocs.platform_nodes import PlatformNodesFactory
```

Comment:
Why using import within function scope?

Reply:
Importing here to avoid cyclic dependency problem